### DISCUSSION TITLE
Error Encountered When Creating Custom GPT Using Provided Schema

### DISCUSSION THREAD

**Discussion by RajaRuling** [coddingtonbear/obsidian-local-rest-api#108] [General]
## Description
I am encountering an error when trying to create a custom GPT model using the provided schema from the Obsidian Local REST API project.

## Steps to Reproduce
Downloaded the Swagger YAML file from [here](https://raw.githubusercontent.com/coddingtonbear/obsidian-local-rest-api/main/docs/openapi.yaml).
- Use the Import from URL in any custom gpt configure section
 
## Expected Behavior
- Successful parsing
- Able to query obsidian notes using rest api actions

## Actual Behavior
The process results in an error. Below is the screenshot of the error message I am receiving:

### Error screenshot 

<img width="602" alt="image" src="https://github.com/coddingtonbear/obsidian-local-rest-api/assets/22155299/a0282cfe-1073-400c-b5f9-d6f35d66a228">

**Comment by coddingtonbear** [comment]
I’m afraid there’s no bug here, just maybe an incompatibility with the tool you’re using.  Feel free to post a pull request if you are able to find a solution, though!
